builtin/providers/google/resource_compute_address.go

```diff
@@ -14,7 +14,12 @@ func resourceComputeAddress() *schema.Resource {
 		Create: resourceComputeAddressCreate,
 		Read:   resourceComputeAddressRead,
 		Delete: resourceComputeAddressDelete,
-
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
```

Comment:
Oh, sorry! Sloppy reading on my part. Your solution totally works, though ideally I'd say we should just fix the resource to use the ID when reading... No matter, doesn't matter for this PR. :)